charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```
@@ -165,8 +165,8 @@ data:
         # This is needed because the kubelets' certificates are not are generated
         # for a specific pod IP
```

We are now going through the API server instead of directly to the kubelet. Can you adapt the other comments as well?
```suggestion
        # This is needed because the api servers' certificates are not generated
        # for a specific pod IP
```